packages/augur-core/source/contracts/Cash.sol

```diff
@@ -137,7 +132,7 @@ contract Cash is ITyped, ICash, ICashFaucet {
 
     function mint(address _target, uint256 _amount) internal returns (bool) {
         balances[_target] = balances[_target].add(_amount);
-        supply = supply.add(_amount);
+        totalSupply = totalSupply.add(_amount);
```

Comment:
Where is this `totalSupply` variable defined?  This contract appears to only derive from some interfaces (not contracts) and you removed the `totalSupply` function and didn't replace it with a `totalSupply` member variable...

Reply:
Out interfaces are actually contracts. Its defined in IERC20.sol